whylog_vim/proxy/__init__.py

```diff
@@ -14,8 +16,10 @@ def __init__(self, editor):
         self.log_reader = LogReaderProxy(log_reader, config, editor, self)
         self.teacher = TeacherProxy(self.teacher_generator(), config, self.editor, self)
         self._state = States.EDITOR_NORMAL
+        self.log_type = None
```

Comment:
it looks like whylog proxy keeps one log type for just the file which has been opened, when the interface that we agreed to was supposed to keep a mapping from many files (which can be open by vim) which can be assigned to many logtypes. So this code would work only if no more than one non-anonymous file type is open. Please change it to an appropriate mapping.